### PR TITLE
fix(fs): fix double fullscreenchange event

### LINKED ISSUE #5685: player fullscreenchange firing twice
In browsers that support the unprefixed fullscreen API (FF 64+ GC71+), the fullscreenchange event is firing twice every time fullscreen is toggled.

https://jsfiddle.net/jbrhq4s1/1/

### LINKED ISSUE #5745: 7.4.2: vjs-fullscreen class not added (regression)
## Description
Player element in fullscreen is missing the `vjs-fullscreen` class.
Test case: https://blacktrash.org/test/videojs/issue5745.html

## Steps to reproduce
1. Go fullscreen
2. Check player element for `vjs-fullscreen` class - the test case demo logs whether it is present.

### Expected
Player element should have `vjs-fullscreen` class in fullscreen mode.

### Actual
Player element does not have `vjs-fullscreen` class in fullscreen mode.

### Error output
If there are any errors at all, please include them here.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
7.4.2 - perhaps a regression caused by https://github.com/videojs/video.js/commit/2bc90a1899aec36ef57b7c2f207e7f7b15042f85

### PR DESCRIPTION
**Pull request by gkatsev**
This reverts the previous fix (2bc90a1)
and also simplifies handling. Seems like this works a lot better while
also ensuring that the vjs-fullscreen class is still on the player.

Fixes #5685, fixes #5745

### CODE REVIEW

**Comment by gkatsev** [comment]
(Looks like maybe would've been easier not to revert it, but it was easier to get to this state this way, so, 🤷‍♂️ )

**Comment by gkatsev** [comment]
@nackd hey, would I be able to recruit you to take a look at this PR for fullscreenchange stuff? The previous fix unfortunately broke the vjs-fullscreen class, so, I made this PR, which actually is a bit simpler than the other one. Seems to work just as well. Would love it if you'll be able to take a look again!

**Comment by blacktrash** [comment]
@gkatsev - thanks, will do. May take me a couple of days though, because of other stuff on my plate, sorry.

**Comment by gkatsev** [comment]
@blacktrash That's fine, just want as many eyes on this as possible :). I hope to be able to release this early next week.

**Comment by gkatsev** [comment]
Thanks for looking @nackd. Interesting that you see the video disappear when exiting fullscreen, I do not see that. Guess more work is needed...

**Comment by nackd** [comment]
@gkatsev, I didn't build the library myself, took it from here where it happens to me too: https://deploy-preview-5756--videojs-docs.netlify.com/test-example/

**Comment by gkatsev** [comment]
That's what it's for :)

Was just able to reproduce it.

**Comment by gkatsev** [comment]
huh, weirdly enough I can't reproduce locally but can reproduce on the netlify build. Guess I'll make sure to verify with netlify before calling it done.

**Comment by gkatsev** [comment]
I think the last commit fixes the disappearing player on exiting fullscreen. Looking at iOS now.

**Comment by gkatsev** [comment]
older iOS with the old style fullscreen API is getting the events as expected. Going to look at an iOS 12 ipad, since it has support for the spec fs api.

**Comment by gkatsev** [comment]
Interestingly, on iOS with the fs api, we get a fullscreenchange event when exiting fs when using pinch-to-exit or using the native button and not when using our fs toggle.

**Comment by gkatsev** [comment]
Looks like trying to make sure that the document handler is removed causes issues, and it isn't strictly necessary.

@nackd can you retest please? Thank you so much!

**Comment by blacktrash** [comment]
@gkatsev - the [test case demo](https://blacktrash.org/test/videojs/issue5745.html) now is at 8d6d00002ad3 and works as expected so far with a bit of cursory testing.

I'll try to keep the test case at the tip of the fs-part-deux branch.

File: src/js/player.js [hunk 1]
```diff
@@ -15,13 +15,13 @@ import * as Dom from './utils/dom.js';
 import * as Fn from './utils/fn.js';
 import * as Guid from './utils/guid.js';
 import * as browser from './utils/browser.js';
-import {IE_VERSION} from './utils/browser.js';
+import {IE_VERSION} from './utils/browser';
```
[the remark this answers is not in the document]
Comment (gkatsev) [reply src/js/player.js]: I'll probably make it so this isn't a change.

**Comment by gkatsev** [comment]
@blacktrash thanks! appreciate any time you spend on this!

**Comment by nackd** [comment]
I gave it a try and it's working fine now as far as I can see. Cheers!

**Comment by gkatsev** [comment]
Awesome, appreciate the help @nackd!

**Pull request merged**